pom.xml

```diff
@@ -26,8 +26,8 @@
 
         <!-- Project Dependencies -->
         <java.version>11</java.version>
-        <pi4j.version>2.0-SNAPSHOT</pi4j.version>
-        <slf4j.version>2.0.0-alpha1</slf4j.version>
```

Comment:
use the most recent one?
<slf4j.version>2.0.0-alpha5</slf4j.version>

or the same as Pi4J?
<slf4j.version>1.7.30</slf4j.version>

Reply:
yep we are using a released version of slf4j, best practice is to not have child dependencies which are alpha, snapshot... in a release